arraycontext/context.py

```diff
@@ -463,6 +464,15 @@ def compile(self, f: Callable[..., Any]) -> Callable[..., Any]:
         """
         return f
 
+    def force_evaluate(self, array: Array) -> Array:
```

Comment:
I wonder whether we should call this `freeze_thaw` instead to avoid inventing new words.